Sources/BraveVPN/BraveVPN.swift

```diff
@@ -650,6 +673,66 @@ public class BraveVPN {
     }
   }
   
+  // MARK: - Promotion
+  
+  /// Editing product promotion order first monthly and yearly after
```

Comment:
Didn't they ask to show the longer option first? I'd need to look at the docs again

Reply:
Fixed under https://github.com/brave/brave-ios/pull/8415/commits/2ffc995bf5e7d4b042512f8eaa9e17fb58588cff